drupal/settings/all.settings.php

```diff
@@ -41,6 +41,19 @@
 $settings['file_private_path'] = 'sites/default/files/private';
 $settings['file_temp_path'] = 'sites/default/files/private/tmp';
 
+// ClamAV settings.
+$clam_mode = getenv('CLAMAV_MODE') ?: 1;
+
+if ($clam_mode == 0 || strtolower($clam_mode) == 'daemon') {
+  $config['clamav.settings']['scan_mode'] = 0;
+  $config['clamav.settings']['mode_daemon_tcpip']['hostname'] = getenv('CLAMAV_HOST') ?: 'redis';
```

Comment:
Typo; should be `clamav` instead of `redis`.

```suggestion
  $config['clamav.settings']['mode_daemon_tcpip']['hostname'] = getenv('CLAMAV_HOST') ?: 'clamav';
```

Strangely enough, when running `ahoy install` locally, the config changes do not seem to apply; the host remains set at `localhost`.